scripts/clone_template.py

```diff
@@ -164,14 +165,32 @@ def main(**kwargs):
         logger.error("VM is not running")
         return 10
 
-    ip, time_taken = wait_for(provider.get_ip_address, [deploy_args['vm_name']], num_sec=1200,
-                              fail_condition=None)
-    logger.info('IP Address returned is {}'.format(ip))
+    try:
+        ip, time_taken = wait_for(provider.get_ip_address, [deploy_args['vm_name']], num_sec=1200,
+                                  fail_condition=None)
+        logger.info('IP Address returned is {}'.format(ip))
+    except Exception as e:
+        logger.exception(e)
+        logger.error('IP address not returned')
+        return 10
```

Comment:
@psav no specific reason, let me know if we need to add different return code there for better traceback?